server/odc-service/src/main/java/com/oceanbase/odc/service/notification/model/RateLimitConfig.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.service.notification.model;
+
+import java.util.concurrent.TimeUnit;
+
+import lombok.Data;
+
+/**
+ * @author liuyizhuo.lyz
+ * @date 2024/1/4
+ */
+@Data
+public class RateLimitConfig {
+
+    private long time = 1;
```

Comment:
As Alibaba JAVA development specifications recommended, we better not set default values for DOs/DTOs

Reply:
OK, I've removed this default value.